docs/sql-ref-syntax-aux-resource-mgmt-add-jar.md

```diff
@@ -33,15 +33,30 @@ ADD JAR file_name
 
 * **file_name**
 
-    The name of the JAR file to be added. It could be either on a local file system or a distributed file system.
+    The name of the JAR file to be added. It could be either on a local file system or a distributed file system or an ivy URL.
+    Apache Ivy is a popular dependency manager focusing on flexibility and simplicity. Now we support two parameter in URL query string:
 
+     * transitive: whether to download dependent jars related to your ivy URL. It is case-sensitive and only take last one if multiple transitive parameters are specified.
+     * exclude: exclusion list when download ivy URL jar and dependent jars.
+
+    User can write ivy URL such as:
+
+      ivy://group:module:version
+      ivy://group:module:version?transitive=[true|false]
+      ivy://group:module:version?exclude=group:module,group:module
```

Comment:
`exclude` seems to be misleading here because `transitive` is false by default. Maybe, shall we revise like the following?

```
ivy://group:module:version
ivy://group:module:version?transitive=[true|false]
ivy://group:module:version?transitive=[true|false]&exclude=group:module,group:module
```

Reply:
> `exclude` seems to be misleading here because `transitive` is false by default. Maybe, shall we revise like the following?
> 
> ```
> ivy://group:module:version
> ivy://group:module:version?transitive=[true|false]
> ivy://group:module:version?transitive=[true|false]&exclude=group:module,group:module
> ```

Done